scipy/stats/_morestats.py

```diff
@@ -2844,17 +2863,18 @@ def levene(*samples, center='median', proportiontocut=0.05):
     ...     ref.null_distribution, bins=bins, density=True, facecolor="C1"
     ... )
     >>> ax.legend(['aymptotic approximation\n(many observations)',
-    ...            'exact null distribution'])
+    ...            'randomized null distribution'])
     >>> lev_plot(ax)
     >>> plt.show()
 
-     >>> ref.pvalue
-    0.4559  # exact p-value
+     >>> ref.pvalue  # randomized test p-value
+    0.4559  # may vary
 
-    Note that there is significant disagreement between the exact p-value
-    calculated here and the approximation returned by `levene` above. For small
-    samples with ties, consider performing a permutation test for more
-    accurate results.
```

Comment:
I'm not sure I want to recommend the permutation test like this because it is not precisely the same test. Strictly speaking, the permutation test is for the null hypothesis that the populations are identical, not just that the variances are equal. It is not a perfect test when the groups differ in something other than variance because those other things could account for the small p-value. (But if the statistic is insensitive to those other things, then it can still be argued to be a good test). Instead, I'd rather just introduce the idea and point to the reference as I'm only qualified to give general statistics information, not recommendations suited to a particular situation.